src/lib/email.js

```diff
@@ -0,0 +1,27 @@
+import sendmail from 'sendmail';
+import StudentStore from '../store/student-store';
+
+const mailer = sendmail({
+    silent: true
+});
+
+export function sendMail(emails){
+    return new Promise((res, rej) => {
+        mailer({
+            from: 'testing@msoe.edu',
```

Comment:
I was just going to leave that as is and go back to set the details after we talk to Jim, that goes for most of the comments in the review, but I'll go change it now if you feel it's important to do right away.